### PR TITLE
Jcm pages

### PR DESCRIPTION
**Pull request by cmcfadzean**
## Description
Now that we have our design Principles this PR creates an area for us to take one more step and showing some of these principles in practice. These are experiences that we believe should be consistent across the app. These guidelines will help us become intentional about how we lean into our design Principles in the actual experience we build.

### CODE REVIEW

File: app/views/pages/crud.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<%= content_for :heading do %>
+<%= md(%(
+# Create & Distroy
```
Comment (pixelflips) [inline app/views/pages/crud.html.erb]: Should it be Destroy?

**Review by pixelflips — APPROVED** [review]
Noticed one small typo, but overall an awesome addition. 🔥 🔥

File: app/views/application/_sidebar.html.erb [hunk 2]
```diff
@@ -39,6 +39,38 @@ content_pages = [
 ]
 %>
 <%
+experiences_pages = [
+  {
+    name: "Empty States",
```
Comment (teenwolfblitzer) [inline app/views/application/_sidebar.html.erb:44]: Is there or should there be a particular order here? If not, I'm wondering if this and other menu items could default to alphabetical order, mirroring the elements/objects listings.

File: app/views/pages/null.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<%= content_for :heading do %>
+<%= md(%(
+# Empty States
+<p class="docs-heading__lead">How an experience looks without items to display is as important as how it looks with content.</p>
```
Comment (teenwolfblitzer) [inline app/views/pages/null.html.erb:4]: Surprisingly, "empty states" is the only page with content 🙈

**Review by teenwolfblitzer — APPROVED** [review]
👍 LGTM, had one general question about how we order pages in the nav, good to go once the typo @pixelflips mentioned is corrected

File: app/views/application/_sidebar.html.erb (hunk 2 above)
Comment (cmcfadzean) [reply app/views/application/_sidebar.html.erb:44]: The general idea I had here was tracking the way states would exist in the admin. Empty > then onboarding them > creating > loading during creation.

File: app/views/pages/null.html.erb (hunk 3 above)
Comment (cmcfadzean) [reply app/views/pages/null.html.erb:4]: I love this so much lol

**Pull request merged**